app/src/main/java/com/example/leagueapp/UI/MainActivity.kt

```diff
@@ -0,0 +1,65 @@
+package com.example.leagueapp.UI
+
+import android.app.Fragment
+import android.support.design.widget.BottomNavigationView
+import android.support.v7.app.AppCompatActivity
+import android.os.Bundle
+import android.support.design.internal.BottomNavigationItemView
+import android.support.design.internal.BottomNavigationMenu
+import android.support.design.internal.BottomNavigationMenuView
+import android.util.Log
+
+import com.example.leagueapp.R
+import java.lang.reflect.Field
+import kotlin.reflect.KClass
+
+class MainActivity : AppCompatActivity() {
+
+    override fun onCreate(savedInstanceState: Bundle?) {
+        super.onCreate(savedInstanceState)
+        setContentView(R.layout.activity_main)
+
+        //Create BottomNavigationView, disable shift mode, then set item selected listener
+        val bottomNavigationView = findViewById(R.id.navigation) as BottomNavigationView
```

Comment:
in Kotlin you don't need to do this anymore.  Just access BottomNavigationObject with the id ("navigation"). 
 E.g. navigation.setVisibility(View.GONE) 

Kotlin automagically does the findViewById for you